react-dropzone/react-dropzone.d.ts

```diff
@@ -1,49 +1,42 @@
 // Type definitions for react-dropzone
-// Project: https://github.com/paramaggarwal/react-dropzone
-// Definitions by: Mathieu Larouche Dube <https://github.com/matdube>
+// Project: https://github.com/okonet/react-dropzone
+// Definitions by: Mathieu Larouche Dube <https://github.com/matdube>, Ivo Jesus <https://github.com/LynxEyes>, Luís Rodrigues <https://github.com/goblindegook>
 // Definitions: https://github.com/Vooban/DefinitelyTyped
 
 ///<reference path='../react/react.d.ts' />
 
 declare namespace ReactDropzone {
     import React = __React;
-    interface DropzoneProps {
-        onDrop?: Function;
-        onDropAccepted?: Function;
-        onDropRejected?: Function;
-        onDragEnter?: Function;
-        onDragLeave?: Function;
-        style?: Object;
-        activeStyle?: Object;
-        className?: string;
-        activeClassName?: string;
-        rejectClassName?: string;
-        /**
-        * Clicking the <Dropzone> brings up the browser file picker. To disable, set to true.
-        */
-        disableClick?: boolean;
-        /**
-         * Min file size accepted
-         */
-        minSize?: number;
-        /** 
-        * Max file size accepted
-        */
-        maxSize?: number;
-        /**
-        * To accept only a single file, set this to false.
-        */
-        multiple?: boolean;
-        /**
-        * Filters the file types that are valid. It should have a valid MIME type according to input element, for example:
-        * application/pdf
-        * image/*
-        * audio/aiff,audio/midi
-        */
-        accept?: string;
+    interface IDropzoneProps {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html#naming-conventions
> In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.